deepinv/physics/mri.py

```diff
@@ -115,7 +115,7 @@ def sample_mask(self, image_size=(320, 320), acceleration_factor=4, seed=None):
             image_size[0] // 2 - num_lines_center // 2,
             image_size[0] // 2 + num_lines_center // 2 + 1,
             steps=50,
-            dtype=torch.int,
+            dtype=torch.long,
```

Comment:
Just curious, why?

Reply:
it was giving errors when running the pytest on a gpu